### PR TITLE
Second draft

### PR DESCRIPTION
**Pull request by smpark7**
This is my second draft for review.

Addresses #3

### CODE REVIEW

File: abstract.tex [hunk 1]
```diff
@@ -31,17 +31,35 @@
 \begin{document}
 
 \begin{abstract}
-Molten salt reactors (MSR) differ greatly from conventional solid-fuelled reactors, particularly in the neutronics and thermal-hydraulics behaviours. They present unique computational challenges that conventional computational tools fail to address effectively. The development of novel simulation tools for MSRs is an essential step towards understanding the behaviour of MSRs under various conditions. An important requirement for new MSR simulation tools is strong coupling between neutronics and thermal-hydraulics modeling due to the strong temperature reactivity coefficients arising from Doppler effects and changes in the fuel salt density.
+%
+This abstract presents the outline for the code-to-code verification of Moltres.
```
Comment (katyhuff) [inline abstract.tex]: What does it mean for an abstract to present an *outline* of verification?

File: abstract.tex [hunk 2]
```diff
@@ -31,17 +31,35 @@
...
+Moltres \cite{lindsay_introduction_2018} is an open source coupled neutronics/thermal hydraulics simulation tool for simulating molten salt reactors (MSR).
```
Comment (katyhuff) [inline abstract.tex]: consider using the glossaries package for auto-expanding acronyms.

File: abstract.tex [hunk 3]
```diff
@@ -31,17 +31,35 @@
...
+MSRs differ greatly from conventional solid-fuelled reactors, particularly in their neutronics and thermal-hydraulics behaviours.
+They present unique computational challenges that many existing computational tools fail to address effectively.
+Strong coupling between neutronics and thermal-hydraulics modeling is an important requirement for new MSR simulation tools due to the strong temperature reactivity coefficients arising from Doppler effects and changes in the fuel salt density.
```
Comment (katyhuff) [inline abstract.tex]: Sentence is too convoluted. 
```suggestion
MSR simulation tools must capture strong coupling between neutronics and thermal-hydraulics exhibited by delayed neutron precursor movement as well as strong Doppler and density feedbacks in the fuel salt.
```

File: abstract.tex [hunk 4]
```diff
@@ -31,17 +31,35 @@
...
 
-Moltres is an open source coupled neutronics/thermal hydraulics simulation tool for simulating MSRs \cite{lindsay_introduction_2018}. It is under active development at the University of Illinois. It is an application code built using the MOOSE finite element framework \cite{gaston_moose:_2009}. Moltres is developed as a solver for the coupled multi-group neutron diffusion, temperature and delayed neutron precursor governing equations. As a multi-group neutronics solver on a coarse mesh, Moltres imposes relatively low computational loads compared to continuous energy neutronics solvers and computational fluid dynamics thermal-hydraulic solvers employed by other researchers in the study of MSRs. This allows for steady-state, transient and accident analyses with relatively low computational requirements.
+The development of novel simulation tools for MSRs is an essential step towards understanding the behaviour of MSRs under various conditions, one of which is the changing fuel compositions across the lifespan of MSRs.
```
Comment (katyhuff) [inline abstract.tex]: unnecessary passive voice.

File: abstract.tex [hunk 5]
```diff
@@ -31,17 +31,35 @@
...
+The fuel composition in an MSR continuously evolves over time as online refueling replenishes fissile material while removing fission product poisons. The impact of changing fuel composition on the safety parameters such as the reactivity feedback coefficient must be studied extensively.
```
Comment (katyhuff) [inline abstract.tex]:
```suggestion
The fuel composition in an MSR continuously evolves over time as online refueling replenishes fissile material while removing fission product poisons. The impact of changing fuel composition on the safety parameters such as the reactivity feedback coefficient must be fully characterized to establish a licensing case for this class of reactors.
```

File: abstract.tex [hunk 6]
```diff
@@ -31,17 +31,35 @@
...
 
-This abstract presents the outline for the code-to-code verification of Moltres. The reactor model used for this verification is the Molten Salt Fast Reactor. The MSFR is a reference design for a fast-spectrum molten salt reactor developed under the EVOL and SAMOFAR projects\cite{serp_molten_2014}. There is abundant MSFR parameter and simulation data available from various authors \cite{fiorina_modelling_2014} \cite{pettersen_coupled_2016} for the reference model to perform the code-to-code verification.
+Moltres is an application code built in the MOOSE finite element framework \cite{gaston_moose:_2009}.
```
Comment (katyhuff) [inline abstract.tex]: - unnecessary passive voice.
- don't use the word "code" in this way.  (application is a noun... there's no reson to add the word code... and "software package" or "computational tool" or similar is preferable to "code" )

File: abstract.tex [hunk 7]
```diff
@@ -31,17 +31,35 @@
...
+Moltres is developed as a solver for the coupled multi-group neutron diffusion, temperature and delayed neutron precursor governing equations.
```
Comment (katyhuff) [inline abstract.tex]: unnecessary passive voice.

File: abstract.tex [hunk 8]
```diff
@@ -31,17 +31,35 @@
...
+MOOSE is a parallel framework and it has an adaptive meshing scheme.
```
Comment (katyhuff) [inline abstract.tex]: Unnecessary passive voice. These sentences are also very monotonous when stacked back to back in this way.. "Moltres is... Moltres is.... MOOSE is..." Try reading them aloud and you'll see what I mean..

File: abstract.tex [hunk 9]
```diff
@@ -31,17 +31,35 @@
...
+Therefore, reactor simulations with large complex meshes can scale effectively when run on multiple cores.
```
Comment (katyhuff) [inline abstract.tex]: This sentence is a bit of a "so what". (connect this sentence to the topic of your abstract.)

File: abstract.tex [hunk 10]
```diff
@@ -31,17 +31,35 @@
...
+Moltres is an application code built in the MOOSE finite element framework \cite{gaston_moose:_2009}.
+Moltres is developed as a solver for the coupled multi-group neutron diffusion, temperature and delayed neutron precursor governing equations.
+MOOSE is a parallel framework and it has an adaptive meshing scheme.
+Therefore, reactor simulations with large complex meshes can scale effectively when run on multiple cores. 
 
-Moltres requires neutron group constant data from a neutron transport code. For this study, Serpent is used to generate the group constant data. Serpent \cite{leppanen_serpent_2015} is a continuous-energy Monte Carlo code used in numerous reactor physics applications. The inputs consist of MSFR reference model parameters, neutron energy group bounds, a nuclear data library, and the temperatures at which the group constants are to be generated. Six neutron group constant data is generated for the six-group neutron diffusion equation.
+The Molten Salt Fast Reactor (MSFR) is the MSR model used for the code-to-code verification.
```
Comment (katyhuff) [inline abstract.tex]: Why so much passive voice?
In #2 I expressed that there was too much passive voice in the first draft.
This PR (which was supposed to address #2, not #3 by the way... ) seems to go entirely in the other direction, adding passive voice where active voice used to appear.... What is goign on? I am going to stop reading here. Please fix this before the next request for a review.

**Review by katyhuff — CHANGES_REQUESTED** [review]
Why so much passive voice?
In #2 I expressed that there was too much passive voice in the first draft.
This PR (which was supposed to address #2, not #3 by the way... ) seems to go entirely in the other direction, adding passive voice where active voice used to appear.... What is going on? I stopped reading halfway through because it's clear my previous comment was not addressed. Please fix this before the next request for a review.

**Comment by smpark7** [comment]
I apologize for the passive voice sentences. I was trying to remove them but I failed to recognize them and inadvertently added more.

**Comment by smpark7** [comment]
The deadline has been extended until 15 march.

**Comment by katyhuff** [comment]
I will merge this and submit corrections with a PR. Please do not submit until that (upcoming) PR is merged.

**Pull request merged**